.github/workflows/land-blocking.yml

```diff
@@ -15,6 +15,7 @@ jobs:
       - name: Setup env
         run: |
           echo "::set-env name=LIBRA_GIT_REV::$(git rev-parse --short=8 HEAD)"
+          echo "::set-env name=TEST_TAG::dev_${LIBRA_GIT_REV}"
```

Comment:
So we set `dev_$hash` here, and the re-doing it in buildspec:21?
Can we use different prefixes here and there?

Reply:
I sometimes use `build-aws.sh` directly when I only need to build images. For those cases the tag in buildspec.yaml generates a canonical tag regardless of what is used in `--version`.

The explicit `TEST_TAG` which is set here ensures that we don't rely on the canonical tag generated by Codebuild. Both of them being the same is no-op. 

If we were to use a different prefix, we need to configure ECR Lifecycle Policy (https://github.com/calibra/libra-ops/blob/93a1f1813d3baec908e5d81750c43d536c2160ff/terraform/global-dev/ecr.tf#L126) to GC them. I don't see any value in a different prefix right now.

Reply:
Value here is less confusion - right now it seems that sometimes it going to tag image with two tags if --version and default does not match, or one tag if they match. Also whoever uses this later will not understand where does dev_xxx coming from, since it is effectively coming from two places. Looks like lifecycle policy is a one line change, so it would not hard to add exception there.

I guess ultimately it's up to you, but to me current setup seem a bit confusing

@bors-libra delegate+

Reply:
CC @sausagee for his opinion since he setup the tag management for images.

I'm ok with adding a `land_blocking_` prefix, but I feel it's less confusing if we have lesser patterns of tags in the repo.

Reply:
`land_blocking_xxx` is fine.  I'd suggest removubg `LIBRA_GIT_REV` in the workflow and replace it with `TEST_TAG`.

Reply:
Scratch that.  It looks like you did that already.  

@bors-libra delegate+

Reply:
I think we need both because LIBRA_GIT_REV is the `hash` and TEST_TAG is `dev_hash`